BUGBOUNTY-PREVIEW-SCOPES.md

```diff
@@ -0,0 +1,33 @@
+# .NET Bounty Scope
+
+## Introduction
+
+The [.NET bug bounty](https://www.microsoft.com/en-us/msrc/bounty-dot-net-core) covers current released versions of .NET Core, ASP.NET Core and .NET, as well as the latest beta or release candidate of the upcoming version.
+
+.NET has the concept of preview features which ship in the current version, or in nightly builds of the upcoming version. Preview features are not enabled by default, but their goal is to be enabled in the next major release. Preview features are now considered in scope for the bug bounty if they are listed in the table below.
+
+Bugs against preview features in the current release must be demonstrated against the latest current release.
```

Comment:
Let's assume for the sake of argument that HTTP/3 support ships proper in .NET 7. How would the "current" release play out in the table below? Do we say that as soon as the feature goes stable and vNext ships, it's now unsupported (thus ineligible for bounty) in .NET 6?